packages/dashboard/e2e/README.md

```diff
@@ -0,0 +1,36 @@
+# E2E workflow
+
+This is to document the flow and interaction of the e2e services in the github environment when:
+
+- starting up the services
+- running the tests
+
+## Starting up the services
+
+Below is a diagram representing the flow of commands when running `npm run test:e2e`
+
+![Flow of commands diagram](documentation-utils/e2e-E2e-start-process.png)
```

Comment:
Might want to change to diagram to show that `npm run test:e2e` runs `source env.sh` -> `npm run setup`... sequentially. The diagram seems to suggest they are ran in parallel.